src/crwimage_int.cpp

```diff
@@ -37,7 +37,7 @@ constexpr RotationMap::OmList RotationMap::omList_[] = {
 
 uint16_t RotationMap::orientation(int32_t degrees) {
   uint16_t o = 1;
-  for (auto&& [deg, orient] : omList_) {
+  for (auto&& [orient, deg] : omList_) {
```

Comment:
Was this a bug? I am surprised that this is not changing behaviour 😮

Reply:
Yeah it was. Maybe this code is not tested